packages/cli/src/fingerprint/fingerprintWatchCommand.ts

```diff
@@ -63,6 +63,13 @@ export default class FingerprintWatchCommand {
       // Don't crash if too many files are open. When the files close
       // the indexer will pick them up.
       console.warn(error.stack);
+      Telemetry.sendEvent({
+          name: `index:watcher_error:emfile`,
```

Comment:
Did you add this to the telemetry spec spreadsheet?

Reply:
I believe I did.  This is in the telemetry spec spreadsheet under "Events: CLI".